CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed support for UE 4.24.
 - `MaxRPCRingBufferSize` setting has been removed. This was previously used to specify the RPC ring buffer size when generating schema. Now, `DefaultRPCRingBufferSize` is used, and can be overridden per RPC type using `RPCRingBufferSizeOverrides`.
 - `RPCRingBufferSizeMap` setting has been renamed to `RPCRingBufferSizeOverrides`.
+- Changed how FGameplayAbilitySpecHandle are generated to account for multi-worker scenarios. Handles are now only generated as a result of an ability being given to an Ability System Component (ASC), and not when FGameplayAbilitySpecHandle is constructed. Handles are now also only unique among all other handles generated by the same ASC, while in native Unreal they are unique among all handles in the game.
```

Comment:
```suggestion
- Changed how `FGameplayAbilitySpecHandle`s are generated to account for multi-worker scenarios. Handles are now only generated as a result of an ability being given to an Ability System Component (ASC), and not when `FGameplayAbilitySpecHandle` is constructed. Additionally, handles are now only unique among all other handles generated by the same ASC, while in native Unreal they are unique among all handles in the game.
```

Reply:
This is a slight improvement to presentation. I'm still unclear what, as an game dev, I'm supposed to do with this information.
* Why did you make this changes, what's in it for me?
* Is any action required on my part? Do I need to modify my game code?